mmpose/models/heads/voxelpose_head.py

```diff
@@ -18,23 +18,27 @@ class CuboidCenterHead(nn.Module):
     pooling).
 
     Args:
```

Comment:
How are the `CuboidCenterHead` and `CuboidPoseHead` being used in the model config of model? Please complete the model definition.